docs/docusaurus.config.js

```diff
@@ -89,7 +89,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           editUrl:
-            'https://github.com/anitab-org/bridge-in-tech-backend/docs',
+            'https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/docs/docs/Home.md',
```

Comment:
```suggestion
            'https://github.com/anitab-org/bridge-in-tech-backend/edit/develop/docs/docs/Home.md',
```
As It is an edit URL in docusaurus it must redirect to the edit page.